pyiron/vasp/base.py

```diff
@@ -724,6 +748,25 @@ def write_magmoms(self):
         else:
             s.logger.debug("No magnetic moments")
 
+    def set_eddrmm(self, status="not_converged"):
+        """
+        Sets the way, how EDDRMM warning is handled.
+
+        Args:
+            status (str): new status of EDDRMM handling (can be 'not_converged', 'ignore', or 'restart')
```

Comment:
`str:` instead of `status (str):`

Reply:
As an explanation: We are using the Google Style Python Docstrings as explained here https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
yes, agreed, but since 'status' is an argument my way is correct, isn't it?